vcell-core/src/main/java/cbit/vcell/mapping/NetworkTransformer.java

```diff
@@ -781,6 +802,8 @@ private void transform(SimulationContext simContext, SimulationContext transform
 		elapsedTime = endTime - startTime;
 		msg = "Adding " + outputSpec.getObservableGroups().length + " observables to model, " + elapsedTime + " ms";
 		System.out.println(msg);
+		msg = "removeObsTime: " + removeObsTime + ",  addParamTime: " + addParamTime;
+		System.out.println(msg);
 

```

Comment:
remove all System.out or System.err printing ... use Logger instead if you must, or remove if not needed anymore.  Example: lg.debug(msg) when you must have diagnostic messages.

import org.apache.logging.log4j.LogManager; 
import org.apache.logging.log4j.Logger; 
public class NetworkTransformer implements SimContextTransformer { 
      private final static Logger lg = LogManager.getLogger(NetworkTransformer.class);

Reply:
done